src/supplemental/quic/quic_api.c

```diff
@@ -1189,11 +1196,13 @@ quic_pipe_recv_cb(void *arg)
 		nni_mtx_unlock(&qstrm->mtx);
 	}
 
-	if (qstrm->rrlen > 0)
+	if (qstrm->rrlen > 0) {
+		log_debug(" Rein cap: %d, pos: %d, len: %d\n", qstrm->rrcap,
+		    qstrm->rrpos, qstrm->rrlen);
 		if (!nni_list_empty(&qstrm->recvq))
-			nni_aio_finish(&qstrm->rraio, 0, 0);
+			nni_aio_finish_sync(&qstrm->rraio, 0, 0);
```

Comment:
You can't finish rraio in a sync way.
Calling stack overflow would happen when rrlen keeps greater than 0.
Which is easy to happen when the bridge is in a high throughout.